mne/viz/misc.py

```diff
@@ -578,3 +576,92 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
     if show:
         fig.show()
     return fig
+
+
+def plot_phase_amplitude_coupling(phase_bins, normalized_amplitude,
+                                  title=None, show=True):
+    """Plot the phase amplitude plot.
+
+    Parameters
+    ----------
+    phase_bins : array, shape (number of bins + 1,)
+        Binned phase time series of phase modulating signal.
+    normalized_amplitude : ndarray, shape (n_epochs, bin_num)
+        The normalized amplitude values across each bin.
+    title : str, None
+        Title, default is 'Phase amplitude plot'.
```

Comment:
Cool, also going to have it return the figure so you can do other stuff with the plot if need be. Right now it wasn't returning anything.